huddlyproto/proto/huddly.proto

```diff
@@ -0,0 +1,231 @@
+syntax = "proto3";
```

Comment:
Some stuff missing in regards to what Geir was working on, but it can also be included in my PR :)

Reply:
We will probably have to rely on one source of truth about the proto file. It is most likely that the proto files will not live here but on falcon-interface instead.

Reply:
Ah ok, was thinking that we might keep it here until the falcon-interface stuff is figured out.